server/tables/tables.go

```diff
@@ -952,32 +952,37 @@ func PostAutoMigrate(db *gorm.DB) error {
 	prefixIndexes, ok := prefixIndicesByDialect[db.Dialector.Name()]
 	if ok {
 		for name, cols := range prefixIndexes {
-			indexes[name] = cols
+			invocationIndices[name] = cols
 		}
 	}
 
-	m := db.Migrator()
-	if m.HasTable("Invocations") {
-		for indexName, cols := range indexes {
-			if m.HasIndex("Invocations", indexName) {
-				continue
-			}
-			err := db.Exec(fmt.Sprintf("CREATE INDEX `%s` ON `Invocations`%s", indexName, cols)).Error
-			if err != nil {
-				log.Errorf("Error creating %s: %s", indexName, err)
-			}
-		}
+	executionIndices := map[string]string{
+		"executions_created_at_usec_index": "(`created_at_usec`)",
+	}
 
-		// Drop deprecated invocation indexes
-		if db.Migrator().HasIndex("Invocations", "invocations_test_grid_query_index") {
-			if err := db.Migrator().DropIndex("Invocations", "invocations_test_grid_query_index"); err != nil {
-				log.Errorf("Error dropping deprecated index: %s", err)
+	indicesByTable := map[string]map[string]string{
+		"Invocations": invocationIndices,
+		"Executions":  executionIndices,
+	}
+
+	m := db.Migrator()
+	for tableName, indexes := range indicesByTable {
+		if m.HasTable("Invocations") {
```

Comment:
This should be `tableName` instead of `Invocations`.

Do you know why we have this check to begin with? I can't think of a circumstance under which these tables would not exist.

Reply:
Looks like Tyler added it in https://github.com/buildbuddy-io/buildbuddy/commit/f35cf8c0e411f1cab7371180b411ec054928ce8b but I'm not sure why :shrug:

Reply:
Done.